index.bs

```diff
@@ -1912,8 +1924,12 @@ partial interface MLGraphBuilder {
                     1. If |dim| is equal to |axis|, add to |desc|.{{MLOperandDescriptor/dimensions}}[|axis|] the value of |inputs|[|index|].{{MLOperandDescriptor/dimensions}}[|dim|].
         1. Let |output| be the result of invoking the <a>create MLOperand</a> steps given [=this=] and |desc|.
             1. If that throws an error, re-throw the error and stop.
-        1. Make a request to the underlying platform to create an operator for this method with |inputs| connected as input and |output| connected as output and store a reference to the [=implementation-defined=] platform object to |output|.{{MLOperand/[[operand]]}}.
-            1. If that fails, throw a "{{DataError}}" {{DOMException}} and stop.
+        1. If any of the following sub-steps fail, throw an "{{OperationError}}" {{DOMException}} and stop.
+            1. Make a request to the underlying platform to:
+                1. Create an [=implementation-defined=] platform operator |concat| for this method, given |inputs| and |axis|.
+                1. Register |inputs|.{{MLOperand/[[operand]]}} as an input to |concat|.
```

Comment:
concat may have multiple inputs.

Reply:
concatImpl? be consistent with other ops?

Reply:
Yes, yes. Multiple rounds of alignment went missing this one. :)